### ISSUE TITLE
Website is crashing

### ISSUE THREAD

**Issue by ashnehete** [kubernetes-sigs/release-notes#691]
It is because the [release-notes-index.json](https://cdn.dl.k8s.io/release/release-notes-index.json) pattern has changed. Which we relied on to fetch all the remote release notes. I'm still trying to figure out why it changed and how to fix it, but opening this issue to start the conversation.

**Comment by ashnehete** [comment]
The file is storing gibberish URLs at the moment which seems to be fixed with the latest one:
```json
  "v1.30.0-rc.1": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.30.0-rc.1/release-notes.json",
  "v1.30.0-rc.2": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.30.0-rc.2/release-notes.json",
  "v1.30.1": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.30.1/release-notes.json",
  "v1.30.2": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.30.2/release-notes.json",
  "v1.30.3": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.30.3/release-notes.json",
  "v1.30.4": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.30.4/release-notes.json",
  "v1.30.5": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.30.5/release-notes.json",
  "v1.30.6": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.30.6/release-notes.json",
  "v1.31.0": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.31.0/release-notes.json",
  "v1.31.0-alpha.1": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.31.0-alpha.1/release-notes.json",
  "v1.31.0-alpha.2": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.31.0-alpha.2/release-notes.json",
  "v1.31.0-alpha.3": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.31.0-alpha.3/release-notes.json",
  "v1.31.0-beta.0": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.31.0-beta.0/release-notes.json",
  "v1.31.0-rc.0": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.31.0-rc.0/release-notes.json",
  "v1.31.0-rc.1": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.31.0-rc.1/release-notes.json",
  "v1.31.2": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.31.2/release-notes.json",
  "v1.32.0-alpha.1": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.32.0-alpha.1/release-notes.json",
  "v1.32.0-alpha.2": "https://storage.googleapis.com/https://storage.googleapis.com/https://dl.k8s.io/release/v1.32.0-alpha.2/release-notes.json",
  "v1.32.0-alpha.3": "https://storage.googleapis.com/https://dl.k8s.io/release/v1.32.0-alpha.3/release-notes.json",
  "v1.32.0-beta.0": "https://dl.k8s.io/release/v1.32.0-beta.0/release-notes.json"
```

**Comment by ashnehete** [comment]
Opened issue in https://github.com/kubernetes/release/issues/3833

**Comment by ashnehete** [comment]
Once this is fixed, there is also the issue of CORS not enabled over `dl.k8s.io`. That is the reason we went with `cdn.dl.k8s.io` before.

**Comment by Vyom-Yadav** [comment]
Thanks for creating these issues. The CORS discussion is going over here: https://kubernetes.slack.com/archives/CCK68P2Q2/p1731628311461459

**Comment by saschagrunert** [comment]
I have a fix in #700
